doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -84,50 +84,51 @@ feature-gates:
 - container-d (with CRI plugin enabled, which it is by default)
 - Docker CE (via the dockershim, which is currently built in to the kubelet)
 - CRI-O
 - Frakti
 
-To support `req.sec.gen.04` the architecture specifies the usage of a container runtime with the capability of Kernel isolation:
+To support `req.inf.vir.01` the architecture specifies the usage of a container runtime with the capability of Kernel isolation:
 - kata-containers
 
-These specifications cover the [full lifecycle of a container](https://github.com/opencontainers/runtime-spec/blob/master/runtime.md#lifecycle) `creating > created > running > stopped` which includes the use of storage that is required during this lifecycle - this is management of the Host OS filesystem by the container runtime. This lifecycle management by the container runtime (when conformant with the above specifications) supports the requirement `req.inf.stg.06` for ephemeral storage for Pods.
+These specifications cover the [full lifecycle of a container](https://github.com/opencontainers/runtime-spec/blob/master/runtime.md#lifecycle) `creating > created > running > stopped` which includes the use of storage that is required during this lifecycle - this is management of the Host OS filesystem by the container runtime. This lifecycle management by the container runtime (when conformant with the above specifications) supports the ephemeral storage for Pods.
 
-To fulfill `req.sec.gen.05` the architecture specifies the usage of the Kubernetes CPU Manager and to support the isolation of workload resources from the infrastructure resources [CPU Pooler](https://github.com/nokia/CPU-Pooler/).
+To support the isolation of the resources used by the infrastructure from the resources used by the workloads the architecture specifies the usage of the Kubernetes CPU Manager and [CPU Pooler](https://github.com/nokia/CPU-Pooler/).
 
 > Todo: details and RA2 specifications relating to runtimes in order to meet RM features and requirements from RM chapters 4 and 5.
 
 <a name="4.5"></a>
 ## 4.5 CNI plugins
 
-> Editors note: The following chapter lists a set of CNI plugins conformant with the Reference Architecture. In future releases the list of CNI plugins should be refined in a way that there is only one example component selected for each functionality. 
-
-The selected CNI multiplexer/metapulgin may be [DANM](https://github.com/nokia/danm) as it provides the possibility to use several other CNI plugins (`req.inf.ntw.16`) and provides an API based solution to administer the networks (`req.inf.ntw.10`) from a central point (`req.inf.ntw.11`).<br>
+As the selected CNI multiplexer/metapulgin MUST support other CNI plugins (`req.inf.ntw.06`) and SHOULD provide an API based solution to administer the networks from a central point (`req.inf.ntw.03`) the selected CNI multiplexer/metapulgin may be [DANM](https://github.com/nokia/danm).<br>
```

Comment:
Suggested update to wording:

> The selected CNI multiplexer/metapulgin MUST be conformant with the [CNI spec](https://github.com/containernetworking/cni/blob/master/SPEC.md), MUST support other CNI plugins (`req.inf.ntw.06`), and SHOULD provide an API based solution to administer the networks from a central point (`req.inf.ntw.03`).  An example CNI implementation which meets these requirements is [DANM](https://github.com/nokia/danm).

Reply:
Alternative wording:

The selected CNI multiplexer/metapulgin MUST be conformant with the [CNI spec](https://github.com/containernetworking/cni/blob/master/SPEC.md), MUST support other CNI plugins (`req.inf.ntw.06`), and SHOULD provide an API based solution to administer the networks from a central point (`req.inf.ntw.03`).  Example CNI implementations which meet these requirements include:
- [DANM](https://github.com/nokia/danm/#danm)
- [Multus](https://github.com/intel/multus-cni#multus-cni)

Reply:
It may make sense to have a section discussing the APIs which are provided by additional items outside of the CNI. Even in the DANM project the API support is not part of the CNI plugin itself it is additional components (eg. netwatcher, svcwatcher.  see [multus doc](https://github.com/nokia/danm/#install-an-akraino-rec-and-get-danm-for-free)  for more)-- though I'm not clear if the DANM CNI plugin can be used w/o the other components independently.

Reply:
@taylor Thanks. Also, if we look at the direction stated in the [DANM documen](https://github.com/nokia/danm/#install-an-akraino-rec-and-get-danm-for-free)t, "Fakeipam is a temporary solution, the long-term aim is to separate DANM's IPAM component into a full-fledged, standalone IPAM solution."